.eslintrc.js

```diff
@@ -1,7 +1,23 @@
 module.exports = {
-    "extends": "standard",
-    "plugins": [
-        "standard",
-        "promise"
-    ]
-};
\ No newline at end of file
+  'extends': 'standard',
+  'plugins': [
+    'standard',
+    'promise'
+  ],
+  'overrides': [
+    {
+      'files': ['**.test.js'],
+      'rules': {
+        'no-unused-expressions': 'off'
+      }
+    }
+  ],
+  'globals': {
+    'test': true,
+    'expect': true,
+    'afterAll': true,
+    'beforeAll': true,
+    'afterEach': true,
+    'beforeEach': true
```

Comment:
U may use https://www.npmjs.com/package/eslint-plugin-jest instead of doing all of that manually here.